README.md

```diff
@@ -1,130 +1,146 @@
-# ![nf-core/rnafusion](docs/images/nf-core-rnafusion_logo.png)
+# ![nf-core/rnafusion](docs/images/nf-core/rnafusion_logo_light.png#gh-light-mode-only) ![nf-core/rnafusion](docs/images/nf-core/rnafusion_logo_dark.png#gh-dark-mode-only)
```

Comment:
```suggestion
# ![nf-core/rnafusion](docs/images/nf-core-rnafusion_logo_light.png#gh-light-mode-only) ![nf-core/rnafusion](docs/images/nf-core-rnafusion_logo_dark.png#gh-dark-mode-only)
```

Reply:
That's fixed in nf-core tools dev, good catch, I always forgot to check this one out